docs/custom-flows/email-password.mdx

```diff
@@ -817,7 +816,7 @@ To authenticate a user using their email and password, you must:
   </Tab>
 
   <Tab>
-    First create a route group with a layout that redirects users if they're already signed in, as shown in the following example:
+    Create a route group with a layout that redirects users if they're already signed in, as shown in the following example:
```

Comment:
Seems like we don't need to mention this again, as we also mention this above in the sign-up flow